contracts/factory/src/contract.rs

```diff
@@ -188,6 +188,10 @@ impl FactoryTrait for Factory {
         )
             .into_val(&env);
 
+        if let PoolType::Xyk = pool_type {
+            init_fn_args.push_back(100i64.into_val(&env));
```

Comment:
But now you've hardcoded it for the factory. It should becoma a parameter during the pool creations.

Reply:
jeez, I guess I thought I was inside the test setup. Fixed now